prime-router/src/main/kotlin/azure/RequestFunction.kt

```diff
@@ -70,27 +73,40 @@ abstract class RequestFunction(
         routeTo.filter { workflowEngine.settings.findReceiver(it) == null }
             .forEach { actionLogs.error(InvalidParamMessage("Invalid receiver name: $it")) }
 
+        var sender: Sender? = null
+        var schema: Schema? = null
         val clientName = extractClient(request)
         if (clientName.isBlank()) {
-            actionLogs.error(InvalidParamMessage("Expected a '$CLIENT_PARAMETER' query parameter"))
-        }
-
-        val sender = workflowEngine.settings.findSender(clientName)
-        if (sender == null) {
-            actionLogs.error(InvalidParamMessage("'$CLIENT_PARAMETER:$clientName': unknown sender"))
-        }
-
-        // verify schema if the sender is a topic sender
-        var schema: Schema? = null
-        if (sender != null && sender is HasSchema) {
-            schema = workflowEngine.metadata.findSchema(sender.schemaName)
-            if (schema == null) {
+            // Find schema via SCHEMA_PARAMETER parameter
```

Comment:
Note: You will need to refactor this as some point as this only supports the covid pipeline.

Reply:
Noted, created https://github.com/CDCgov/prime-reportstream/issues/8073 to track this work.